teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>With <strong>protonate </strong>I also associate introducing charges if required at a certain pH. How about renaming it to <strong>add_ligand_hydrogens</strong>. Also mention that we are generating charge neutral ligands with this function.</p><p> </p><p>Ouhhh, what happens actually if you pass a SMILES string with a charged atom? Will we get a charged structure from that? This would be great. In this case we could keep the name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>